README.md

```diff
@@ -26,3 +26,23 @@ A sample project is available on GitHub: https://github.com/SonarSource/sonar-ex
 
 ### Run an Analysis with the other Analyzers
 Maven and Ant can also be used to launch analysis on CSS projects.
+
+## Metrics
+### Functions
+Number of rules.
+
+### Complexity
+The following elements increment the complexity by one:
+
+* Class selector
+* ID selector
+* Attribute selector
+* Type selector
+* Pseudo-class selector
+* At-rule
+
+### Complexity/function
+It computes the complexity/rule, meaning the average number of selectors per rule.
+
+It gives a measurement on how specific the selectors are.
+
```

Comment:
Fixed in https://github.com/racodond/sonar-css/commit/2df7c1171ba70b8614c197642ac39df61e38df13